CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixes increase/decrease font-size keybindings on international keyboards
 - On Wayland, the `--title` flag will set the Window title now
 - Parsing issues with URLs starting in the first or ending in the last column
+- Fix not starting maximized on X11
```

Comment:
I think we can improve this a little bit. Right now people might think this means Alacritty will always start maximized, which is not the case.

So we should say that this fixed not starting maximized on X11 when the config setting is enabled.

Reply:
Alright, changed that